include/openthread/platform/radio.h

```diff
@@ -334,7 +334,8 @@ typedef struct otRadioFrame
             /**
              * The timestamp when the frame was received in microseconds.
              *
-             * The value SHALL be the time when the SFD was received when TIME_SYNC or CSL is enabled.
+             * The value SHALL be the time when the SFD was fully received when TIME_SYNC or CSL is enabled.
+             * (That is, the time that marks the end of reception of the last symbol of the SFD.)
              * Otherwise, the time when the MAC frame was fully received is also acceptable.
```

Comment:
Since we have a change to platform header, we need to increment the `OPENTHREAD_API_VERSION` in `intsnace.h`.

GH action "API Version" checks this -> https://github.com/openthread/openthread/actions/runs/4492127980/jobs/7901580504?pr=8894

Reply:
Done; this sounds like one of the SW developer bedtime stories "... and then the automated tests became so complex that even changing a comment line would fail the tests ... "  ;-)